raycicmd/bk_pipeline.go

```diff
@@ -86,7 +86,7 @@ func makeRayWindowsDockerPlugin(config *stepDockerPluginConfig) map[string]any {
 
 	m := map[string]any{
 		"image":          windowsBuildEnvImage,
-		"shell":          []string{"bash", "-c"},
+		"shell":          []string{"bash", "-eco", "pipefail", "--"},
```

Comment:
I think it needs to be `bash -eo pipefail -c`, and without the `--`.

The `-c` will need to take the next arg as input.